webpack/scenes/Subscriptions/SubscriptionValidations.js

```diff
@@ -1,7 +1,7 @@
 import { translate as __ } from 'foremanReact/common/I18n';
 import { filterRHSubscriptions } from './SubscriptionHelpers.js';
 
-export const validateQuantity = (quantity, availableQuantity) => {
+export const validateQuantity = (quantity, upstreamAvailable, maxQuantity) => {
```

Comment:
>It's how we determine whether or not to tell the user that they have exceeded max quantity.

That conditional you linked serves to display 'Exceeds available quantity' if it's true. Since we already have the maxQuantity it seems to me like the upstreamAvailable serves no purpose in `validateQuantity`.

>If the upstream available is less than 0, then it's unlimited and we don't give them the message.

The if statement is checking for >= 0 which may change your assessment of the code!